sensor/common/compliance/auditlog_manager_test.go

```diff
@@ -300,17 +310,17 @@ func (s *AuditLogCollectionManagerTestSuite) TestStateSaverSavesFileStates() {
 	for node := 0; node < 2; node++ {
 		for i := 0; i < 2; i++ {
 			nodeName := fmt.Sprintf("node-%d", node)
-			ts := s.getAsProtoTime(startTime.Add(time.Duration(i*10) * time.Minute))
-			msg := s.getMsgFromCompliance(nodeName, ts)
-			state := s.getAuditLogFileState(ts, msg.GetAuditEvents().Events[0].Id)
+			timestamp := startTime.Add(time.Duration(i*10) * time.Minute)
```

Comment:
nitpicking: The variable name is a bit misleading because it contains `time.Time` and not `Timestamp`.